index.html

```diff
@@ -4,31 +4,42 @@
   @import url('https://fonts.googleapis.com/css2?family=Open+Sans&display=swap');
 
   #wheel {
-    position:absolute;
-    top:0px;
-    left:0px;
-    width:100vmin;
-    height:100vmin;
+    position: relative;
+    top: 0px;
+    left: 0px;
+    width: 100vmin;
+    height: 100vmin;
   }
   .letter {
-    position:absolute;
-    font-size:10vmin;
+    position: absolute;
+    font-size: 10vmin;
     font-family: 'Open Sans', sans-serif;
   }
   #input {
-    position:absolute;
-    font-size:48pt;
-    top:0;
-    left:0;
-    width:100%;
-    background-color:#00000000;
+    position: absolute;
+    font-size: 48pt;
+    top: 0;
+    left: 0;
+    width: 100%;
+    background-color: #0000;
```

Comment:
Is this because you want a transparent background?

Reply:
yeh, although now that I think about it I don't think it's necessary to explicitly set it to transparent. This line can probably be removed